wordpress-mobile/WordPress-Aztec-Android
# Issue/93 fixing editor space newline issue

Fixes #93 

When we show html in source editor it goes through "pretty-print" feature of Jsoup, which ads spaces and newlines around inline elements, and those spaces sneak back into visual editor.

This PR removes all the newlines around non-block elements, so the source resembles actual text more (same as Calypso). This also spaces from appearing between inline elements where they are not supposed to be. 

To test:

1. - Style text to look something like this: **bold**_italic_
2. - Go to HTML mode.
3. - Switch back to visual editor.
4. - Everything should remain as before.


PS: I'm no so good with regexp, so if anyone know's a better way to dealing with this - let me know!
